.circleci/config.yml

```diff
@@ -149,3 +168,10 @@ workflows:
             branches:
               ignore:
                 - main
+      - build_deploy_simple_api:
+          context:
+            - docker-user
+          filters:
+            branches:
+              ignore:
+                - main
```

Comment:
Hmm, so you're going to publish a new docker image on every single commit, is that what we want?
For example if we submit 2 different PRs touching this sub folder, the docker images on docker-hub will oscillate between each PR on every commit. This could be confusing, esp. since we're tagging all docker images with `latest`. Don't we want instead to publish a docker image on main only?